fenix/app/src/androidTest/java/org/mozilla/fenix/onboarding/view/JunoOnboardingMapperTest.kt

```diff
@@ -22,13 +22,31 @@ class JunoOnboardingMapperTest {
     @Test
     fun showNotificationTrue_pagesToDisplay_returnsSortedListOfAllConvertedPages() {
         val expected = listOf(defaultBrowserPageUiData, syncPageUiData, notificationPageUiData)
-        assertEquals(expected, unsortedAllKnownCardData.toPageUiData(true))
+        assertEquals(expected, unsortedAllKnownCardData.toPageUiData(true, false))
     }
 
     @Test
     fun showNotificationFalse_pagesToDisplay_returnsSortedListOfConvertedPagesWithoutNotificationPage() {
         val expected = listOf(defaultBrowserPageUiData, syncPageUiData)
-        assertEquals(expected, unsortedAllKnownCardData.toPageUiData(false))
+        assertEquals(expected, unsortedAllKnownCardData.toPageUiData(false, false))
+    }
+
+    @Test
+    fun showAddWidgetTrue_pagesToDisplay_returnsSortedListOfAllConvertedPages() {
+        val expected = listOf(defaultBrowserPageUiData, addSearchWidgetPageUiData, syncPageUiData)
+        assertEquals(expected, unsortedAllKnownCardData.toPageUiData(false, true))
+    }
+
+    @Test
+    fun showAddWidgetFalse_pagesToDisplay_returnsSortedListOfConvertedPagesWithoutNotificationPage() {
```

Comment:
nit: could be consolidated with `showNotificationFalse_pagesToDisplay_returnsSortedListOfConvertedPagesWithoutNotificationPage` as `showNotificationFalse_and_showAddWidgetFalse_pagesToDisplay_returnsSortedListOfConvertedPagesWithoutNotificationPage`